bin/alluxio-start.sh

```diff
@@ -52,8 +52,7 @@ check_mount_mode() {
     Mount);;
     SudoMount);;
     NoMount)
-      mount | grep "${ALLUXIO_RAM_FOLDER}" > /dev/null
-      if [[ $? -ne 0 || -z ${ALLUXIO_RAM_FOLDER} ]]; then
+      if [[ $(mount | grep -c ${ALLUXIO_RAM_FOLDER}) != 0 ]]; then
```

Comment:
I've created a PR for this. https://github.com/Alluxio/alluxio/pull/3065 
Can you try that?

Personally, I think it is better to set ALLUXIO_RAM_FOLDER as a mount point since it is actually is when the ramdisk is not mounted manually. alluxio.worker.data.folder is used to set the subdir. So i think it is better to keep it.